src/main/java/com/maxmind/minfraud/response/ScoreResponse.java

```diff
@@ -26,7 +27,7 @@ public ScoreResponse(
             @JsonProperty("warnings") List<Warning> warnings
     ) {
         this.creditsRemaining = creditsRemaining;
-        this.id = id;
+        this.id = id == null ? null : UUID.fromString(id);
```

Comment:
Did you try making the parameter a UUID? It is possible that Jackson will just do the right thing.